spec/component/jobs/fetch_oa_metadata_job_spec.rb

```diff
@@ -64,7 +72,8 @@
           expect(pub.reload.open_access_button_last_checked_at).to be_nil
         end
 
-        it 'updates the publication oa status last checked at and workflow state' do
+        it 'updates the publication oa status last checked at, oa status, and workflow state' do
+          expect(pub.reload.open_access_status).to eq 'closed'
```

Comment:
Should this be 'unknown' too?  I may be misunderstanding, but if we checked Unpaywall and didn't get anything we want to set the oa status to 'unknown', right?

Reply:
Yep, I made a dumb mistake. Both are fixed now